crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -1144,18 +1156,18 @@ impl<F: Clone + Send + Sync> Domain<F, api::PaymentsRequest, PaymentData<F>> for
                 uas_utils::create_new_authentication(
                     state,
                     payment_data.payment_attempt.merchant_id.clone(),
-                    connector_name.to_string(),
+                    connector_mca.connector_name.to_string(),
                     business_profile.get_id().clone(),
                     Some(payment_data.payment_intent.get_id().clone()),
-                    connector_transaction_id,
+                    click_to_pay_mca_id.to_owned(),
                     &authentication_id,
                     payment_data.service_details.clone(),
                     authentication_status,
                 )
                 .await?;
             }
         }
-
+        logger::info!("skipping unified authentication service call since payment conditions {:?}, {:?} are not satisfied", payment_data.payment_attempt.payment_method, business_profile.is_click_to_pay_enabled);
```

Comment:
Nit: This would print something along the lines of:

```text
skipping unified authentication service call since payment conditions card, false are not satisfied
```

Which don't indicate what these values, specifically the `false`, signify. Can we use key-value notation instead?

Something like:

```suggestion
        logger::info!(
            payment_method=?payment_data.payment_attempt.payment_method,
            click_to_pay_enabled=?business_profile.is_click_to_pay_enabled,
            "skipping unified authentication service call since payment conditions are not satisfied"
        );
```